public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -209,17 +210,18 @@ export class SharedPreferences extends PureComponent<Props, State> {
     });
   };
 
-  onLocaleChanged = (locale: string) => {
-    this.setState({ locale });
+  onLocaleChanged = (regionalFormat: string) => {
+    this.setState({ regionalFormat });
 
     reportInteraction('grafana_preferences_locale_changed', {
-      toLocale: locale,
+      toLocale: regionalFormat, // should toLocale also be changed?
```

Comment:
toLocale could be toRegionalFormat, and `grafana_preferences_locale_changed` could be `grafana_preferences_regional_format_changed`

Reply:
yes, i think we should change this 👍

Reply:
Change made!